Dockerfile

```diff
@@ -1,27 +1,84 @@
-FROM node:10
```

Comment:
Why switch away from a standard `node` image to `ubuntu`?  We don't have binary dependencies, do we?  I'd prefer to have our node app run inside a `NODE:lts` image, since that's what we use in CI.

If I'm missing something and you have to use Ubuntu, let me know.

Reply:
When I was updating the Docker file, I checked the Node.js and Docker Hub and I realized that version 12 is a better choice and the only reason that I used version 10 in the first version, was because of the templates that had not been updated. Then I changed it to `From node:12`. When I started building the container, it started installing Linux Debian and I surprised because for the first one it had installed Ubuntu. So, to have a same stack always, regardless of updating the version of Node image, I decided to build the whole container from scratch.

Reply:
If we get rid of `nodemon`, which I suggest below, we shouldn't need to install anything beyond node itself.  It's better if we can use a pre-built image vs. building something on our own, since we don't have to maintain anything that way: they will be the default node images.

Being built on Debian or Ubuntu is really the same thing, Ubuntu is a fork of Debian.  Probably once we get this running, we can switch to use alpine or something even smaller.